src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets

```diff
@@ -0,0 +1,305 @@
+<Project>
+
+  <Target
+    Name="_BlazorCopyFilesToOutputDirectory"
+    DependsOnTargets="BlazorBuild"
+    Inputs="@(BlazorItemSpec)"
+    Outputs="@(BlazorItemSpec->'%(TargetOutputPath)')"
+    AfterTargets="CopyFilesToOutputDirectory">
+
+    <!-- Copy the blazor output files  -->
+    <Copy
+      SourceFiles="@(BlazorItemSpec)"
+      DestinationFiles="@(BlazorItemSpec->'%(TargetOutputPath)')"
+      SkipUnchangedFiles="$(SkipCopyUnchangedFiles)"
+      OverwriteReadOnlyFiles="$(OverwriteReadOnlyFiles)"
+      Retries="$(CopyRetryCount)"
+      RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
+      UseHardlinksIfPossible="$(CreateHardLinksForCopyFilesToOutputDirectoryIfPossible)"
+      UseSymboliclinksIfPossible="$(CreateSymbolicLinksForCopyFilesToOutputDirectoryIfPossible)"
+      Condition="'@(BlazorItemSpec)' != '' and '$(CopyBuildOutputToOutputDirectory)' == 'true' and '$(SkipCopyBuildProduct)' != 'true'">    
+    </Copy>
+
+    <ItemGroup>
+      <BlazorOutputs Include="$(ProjectDir)$(OutputPath)dist/**/*" />
+    </ItemGroup>
+    
+  </Target>
+  
+  <Target Name="_BlazorBuildReport"
+          AfterTargets="_BlazorCopyFilesToOutputDirectory">
+
+    <PropertyGroup>
+      <BlazorBuildStatistics Condition="'$(BlazorBuildStatistics)' == ''">normal</BlazorBuildStatistics>
+      <BlazorBuildStatistics Condition="'$(BlazorBuildStatistics)' == 'true'">high</BlazorBuildStatistics>
+    </PropertyGroup>
+
+    <Message Importance="high" Text="Blazor Build result -> @(BlazorOutputs->Distinct()->Count()) files written to $(ProjectDir)$(OutputPath)dist" />
+    <Message Importance="$(BlazorBuildStatistics)" Text="%(BlazorOutputs.Identity)" />
+    </Target>
+
+  <PropertyGroup>
+    <BlazorBuildDependsOn>
+      GetBlazorBuildConfiguration;
+      ResolveBinaryOutput;
+      BuildBlazorIndexHtml;
+    </BlazorBuildDependsOn>
+  </PropertyGroup>
+
+  <Target Name="BlazorBuild" DependsOnTargets="$(BlazorBuildDependsOn)" AfterTargets="Compile" />
```

Comment:
I'd suggesting finding another name, and dropping the `AfterTargets`. This will already be called  as a dependency of `_BlazorCopyFilesToOutputDirectory`

Reply:
I've renamed it to PrepareBlazorOutput and removed the after targets